lib/plugins/aws/invokeLocal/index.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
Good investigation. Fully agree with that approach and switch to eslint v4 and a fixed template as soon as it is available.